### ISSUE TITLE
Fails to install and update automatically generated debug packages

### ISSUE THREAD

**Issue by h4tr3d** [manjaro/pamac#264]
Hi,

PKGBUILD spec allows to automatically generate `${pkgname}-debug` package that contains split debug info:
```
       options (array)
...skip...
           strip
               Strip symbols from binaries and libraries. If you frequently 
               use a debugger on programs or libraries, it may be helpful
               to disable this option.
...skip...
           debug
               Add the user-specified debug flags (DEBUG_CFLAGS, 
               DEBUG_CXXFLAGS) to their counterpart buildflags as 
               specified in makepkg.conf(5). When used in combination 
               with the ‘strip’ option, a separate package containing the
               debug symbols is created.
```
In short: debug package generates when `strip` and `debug` options pointed together. 

Pamac fails to install and update packages:
* install - there is no any suggestions for debug package installing
* upgrade - (in case when it installed by hand or via yaourt or trizen) fails for main package, because installed debug package strongly depend on main package version.

JFI:
* yaourt installs debug packages without any user iterations 
* ~trizen ask user to install such package after main package installation~ trizen can install debug package but also can't update it because do it in two separate steps.

**Comment by guinux** [comment]
Do you have an example?

**Comment by h4tr3d** [comment]
My packages:
- qtcreator-opt-git
- qtcreator-cmakeprojectmanager2-plugin-opt-git
(both with fixed commit version)

I can prepare some a little package (just main.cpp with "hello world") to avoid long compilation waiting. But can do it only at Monday.

**Comment by guinux** [comment]
I tested with other packages with strip and debug option enabled directly in makepkg.conf.
My tests was successful, pamac installs and upgrade debug packages automatically with the main package.

**Comment by h4tr3d** [comment]
@guinux, strange. My pamac version (Manjaro Stable branch) is 4.3.7. I will try to install and update some package from the AUR with this options enabled in makepkg.conf directly.

**Comment by h4tr3d** [comment]
Check again, I clone `f3` AUR package to `f3-test`. 

Turn on options `debug` and `strip`. Initial install with pamac - success, debug package installed too. But pamac does not provide selection option: debug package installed automatically without any user asking. 

After that I just increase pkgrev by one and upload new package, pamac fail to update it:
https://yadi.sk/i/U7JgPLmM3Kgio8

**Comment by guinux** [comment]
I see the problem. It will be fixed in next release.
Anyway, debug package will still be installed without user interaction because it's an advanced feature already enabled via an option.

**Comment by guinux** [comment]
Fixed in v5.0.0
